test/cases/koji.sh

```diff
@@ -122,9 +135,12 @@ sudo cp \
     /etc/pki/ca-trust/source/anchors/osbuild-composer-tests-ca-crt.pem
 sudo update-ca-trust
 
-greenprint "Restarting composer to pick up new config"
+start_jwt_auth_server
+
+greenprint "Restarting composer, stopping a local worker and starting a remote worker"
 sudo systemctl restart osbuild-composer
```

Comment:
If I read this correctly, this script doesn't modify composer's configuration, can we move this line to a file that actually changes the configuration?

Reply:
It does not, but my reasoning was that it needs to pick up the token from the mock server, once they are started.